lib/duplicate_deleter.rb

```diff
@@ -77,7 +77,8 @@ def call(ids, id_type: 'content_id')
         next
       end
 
-      Indexer::DeleteWorker.new.perform(index_names.first, type_to_delete, ids.first)
+      id_to_delete = results[:results].detect { |a| a[:elasticsearch_type] == type_to_delete }[:_id]
```

Comment:
Are we sure `detect` will find it? In the case it doesn't, will `Indexer::DeleteWorker` handle a `nil` ID to delete?